UnitsNet.Tests/CustomCode/VolumetricHeatCapacityTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by \generate-code.bat.
+//
+//     Changes to this file will be lost when the code is regenerated.
+//     The build server regenerates the code before each build and a pre-build
+//     step will regenerate the code on each local build.
+//
+//     See https://github.com/angularsen/UnitsNet/wiki/Adding-a-New-Unit for how to add or edit units.
+//
+//     Add CustomCode\Quantities\MyQuantity.extra.cs files to add code to generated quantities.
+//     Add UnitDefinitions\MyQuantity.json and run generate-code.bat to generate new units or quantities.
+//
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+// Licensed under MIT No Attribution, see LICENSE file at the root.
+// Copyright 2013 Andreas Gullberg Larsen (andreas.larsen84@gmail.com). Maintained at https://github.com/angularsen/UnitsNet.
+
+using System;
+
+namespace UnitsNet.Tests.CustomCode
+{
+    public class VolumetricHeatCapacityTests : VolumetricHeatCapacityTestsBase
+    {
+        protected override bool SupportsSIUnitSystem => false;
+
+        protected override double JoulesPerCubicMeterKelvinInOneJoulePerCubicMeterKelvin => 1;
+        protected override double KilojoulesPerCubicMeterKelvinInOneJoulePerCubicMeterKelvin => 1E-3;
+        protected override double MegajoulesPerCubicMeterKelvinInOneJoulePerCubicMeterKelvin => 1E-6;
+
+        protected override double BtusPerCubicFootDegreeFahrenheitInOneJoulePerCubicMeterKelvin => 1.491066E-5;
```

Comment:
I've noticed earlier that there are BTU (th) and BTU (IT) units. 
I don't know these units well, should both be included? Should we distinguish them in their unit names?

The one you defined seems to be IT.

> British thermal unitIT per cubic foot degree Fahrenheit:  1.491066e-5
> British thermal unitth per cubic foot degree Fahrenheit:  1.492065e-5

https://www.gordonengland.co.uk/conversion/volheatcap.htm

IT vs th:
https://www.translatorscafe.com/unit-converter/en-US/energy/32-33/Btu%20(IT)-Btu%20(th)/

Reply:
Thanks for cleaning up those files.



**I did indeed pick BTU(IT) and I think that is appropriate.  Reasoning as follows:**

1) I've been in industry for 10 years (in the US) and in school for 7.  Until I added this quantity (and used the same converter you found to check) I had never heard of these 'flavors' of BTUs.  Based on memory of the numbers, I think I've always seen Btu(IT)

2) https://www.gordonengland.co.uk/conversion/energy.htm  This site indicates Btu(IT) is the 'default'.

3) The existing Units.Net definition for `Energy.BritishThermalUnit` is a factor of 1055.05585262 from a Joule.  This means it's a Btu(IT).  Therefore, for the Units.Net Volumetric Heat Capacity BTU to be consistent with energy, it must also be Btu(IT)


Funny you should raise this question because I was debating making an issue for it separately because I went down a rabbit hole trying to figure this out.  I ultimately decided against it because 1) it might be a breaking change, and 2) I think what we have now works for "practical" usage.

The extra confusing part is the default 'calorie' energy unit seems to be the (th) version, _not_ the cal(IT) version (which apparently also exists.  So in Units.Net (and I think "most" of the world) you have cal(th) and BTU(IT); but cal(IT) and Btu(th) also exist, as well as cal(mean) and Btu(mean).


The options for "fixing" this is to either:

Define calories/Btus explicitly with their "flavor" (i.e., IT, th, mean), or
Leave the current ones defined as they are, then add the extra "flavors" in we don't have defined.

The first is confusing, because I bet most people would have no idea what the appropriate unit is to choose (myself included) and it's a breaking change because you're renaming the unit.

The second is also confusing because now you have some cal & BTUs units that are explicitly "flavored" and others that aren't, and they're not consistent with each other.  This is maybe okay?  But doesn't seem worth adding unless someone explicitly needs it.  I certainly don't because again, before last Friday - I had no idea this was a thing.

Reply:
I agree with your reasoning. I believe the same argument has been made before, that IT is the widely used one and Th is the oddity. I also like that we are at least consistent within Units.NET.